ironfish/src/memPool/recentFeeCache.ts

```diff
@@ -125,25 +120,17 @@ export class RecentFeeCache {
     return transactions.reverse()
   }
 
-  getSuggestedFee(percentile: number): bigint {
+  estimateFeeRate(percentile: number): bigint {
     if (this.queue.length < this.numOfRecentBlocks) {
-      return this.defaultSuggestedFee
+      return this.defaultFeeRate
     }
 
     const fees: bigint[] = []
     for (const entry of this.queue) {
-      fees.push(entry.fee)
+      fees.push(entry.feeRate)
     }
 
-    fees.sort((a, b) => {
-      if (a < b) {
-        return -1
-      } else if (a > b) {
-        return 1
-      } else {
-        return 0
-      }
-    })
+    fees.sort((a, b) => (a < b ? -1 : a > b ? 1 : 0))
```

Comment:
Can we use ```fees.sort((a, b) => a - b))```

Reply:
The compareFn for sort needs to return a number, but these are bigints, so we need to work around it 😢

Reply:
I see, so need to convert a-b to number type. Either way looks fine.